### PR TITLE
Clean up netanalysis

### PR DESCRIPTION
**Pull request by fortuna**
- Updated dev instructions
- Made minimum Python version 3.6.9 so I can use the code in colab
- Style fixes from autopep8 and linter
- Updated geo data

### CODE REVIEW

**Comment by fortuna** [comment]
BTW, here is a notebook using the new code: https://www.kaggle.com/vinifortuna/tanzania-shutdown-2020-10-27/

File: netanalysis/traffic/data/fetch_google_traffic.py [hunk 1]
```diff
@@ -31,14 +31,15 @@
...
+
 def main(args):
     if not args.output_dir:
         logging.error("Need to specify output directory")
         return 1
     if not os.path.exists(args.output_dir):
         os.makedirs(args.output_dir)
 
-    report = api.ApiTrafficRepository()  # type: modelTrafficRepository
+    report = api.ApiTrafficRepository()  # type: model.TrafficRepository
```
Comment (ohnorobo) [inline netanalysis/traffic/data/fetch_google_traffic.py:42]: make this type explicit if possible

report: model.TrafficRepository = api.ApiTrafficRepository()

File: netanalysis/traffic/analysis/find_anomalies.py [hunk 2]
```diff
@@ -265,7 +266,7 @@ def main(args):
 
     try:
         all_disruptions = find_all_disruptions(
-            repo, repo.list_regions(), product_id_list)  # type: List[RegionDisruption]
+            repo, repo.list_regions(), product_id_list)  # type: List[model.RegionDisruption]
```
Comment (ohnorobo) [inline netanalysis/traffic/analysis/find_anomalies.py:269]: make this type explicit if possible

all_disruptions: List[model.RegionDisruption] = find_all_disruptions(repo, repo.list_regions(), product_id_list)

**Review by ohnorobo — APPROVED** [review]

File: netanalysis/traffic/analysis/find_anomalies.py (hunk 2 above)
Comment (fortuna) [reply netanalysis/traffic/analysis/find_anomalies.py]: Done

File: netanalysis/traffic/data/fetch_google_traffic.py (hunk 1 above)
Comment (fortuna) [reply netanalysis/traffic/data/fetch_google_traffic.py]: Done

**Comment by ajigang** [comment]
[]()d7934c5f61652391b806add7cfecbf8a42273db2

**Pull request merged**